xtask/src/main.rs

```diff
@@ -80,6 +80,20 @@ mod gencomp {
 
         Ok(())
     }
+
+    #[cfg(test)]
+    mod test {
+        use tempfile::tempdir;
+
+        use super::*;
+
+        #[test]
+        fn generate_completions() {
+            let tmp_dir = tempdir().unwrap();
+
+            copy_completions(tmp_dir.path()).unwrap();
+        }
+    }
```

Comment:
Oh. You want to commit completion files to the repo? What benefit does that bring?

For background: The `cargo xtask gencomp` "test" was originally added in #741 after a `clap` migration caused shell completion file generation to panic. And because of our workflows, the panic went unnoticed until we attempted to cut a release. So the only purpose of this test was to ensure that this does not panic.